### PR TITLE
Pe 832 support drives and folders

### PR DESCRIPTION
**Pull request by agsuy**
- Adds loading conditions
- Loads a script for entities
- Migrates documentation

### CODE REVIEW

File: scripts/.bashrc_patch [hunk 1]
```diff
@@ -1,3 +1,9 @@
-if [[ -z "${NO_SETUP}" ]]; then
-  source entry.sh
+export ENTITIES="/home/node/ardrive-cli/.bats/entities.sh"
```
Comment (matibat) [inline scripts/.bashrc_patch:1]: Consider renaming to `ENTITIES_PATH`

File: scripts/.bashrc_patch [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+
+if [[ -z "${NO_SETUP}" ]] && [[ ! -f "/home/node/ardrive-cli/package.json" ]]; then
+    source /home/node/entry.sh
+fi
+
+if [[ -f "${ENTITIES}" ]] && [[ -f "${WALLET}" ]] && [[ $PUB_DRIVE_ID == '00000000-0000-0000-0000-000000000000' ]]; then
+    source /home/node/ardrive-cli/.bats/entities.sh
```
Comment (matibat) [inline scripts/.bashrc_patch]: Use the constant declared at the top here

File: scripts/.bashrc_patch (hunk 1 above)
Comment (matibat) [reply scripts/.bashrc_patch:1]: ~Also, I'm not sure where to look for that `.sh` file~
_just found it_

**Review by matibat — COMMENTED** [review]
Left some comments

File: scripts/.bashrc_patch (hunk 1 above)
Comment (agsuy) [reply scripts/.bashrc_patch:1]: Maybe ENTITIES_SCRIPT?

File: scripts/.bashrc_patch (hunk 2 above)
Comment (agsuy) [reply scripts/.bashrc_patch]: Fixed

File: scripts/.bashrc_patch (hunk 1 above)
Comment (matibat) [reply scripts/.bashrc_patch:1]: Yes, maybe that's fine. We ain't using that suffix anywhere else :+1:

Comment (agsuy) [reply scripts/.bashrc_patch:1]: Lets keep it then

File: README.md [hunk 3]
```diff
@@ -69,29 +67,33 @@ If current working directory is the repo root, we need to execute:
 
 ```docker cp . ardrive-cli-bats:/home/node/ardrive-cli ```
 
-Now you will have you local environment loaded into Docker.
-Every time you want to load your latest changes, just run the above command again.
-## BATS tests 
+Now you will have you local environment loaded into Docker. You do not need to run ```yarn && yarn build ```again
```
Comment (arielmelendez) [inline README.md]:
```suggestion
Now you will have your local environment loaded into Docker. You do not need to run ```yarn && yarn build ```again
```

File: dockerfile [hunk 4]
```diff
@@ -76,6 +77,11 @@ WORKDIR $HOME
 
 # Set env to tmpfs path
 ENV WALLET /home/node/tmp/wallet.json
+# Set testing ENV variables to sample IDs
+ENV PUB_DRIVE_ID "00000000-0000-0000-0000-000000000000"
+ENV PUB_FOLD_ID "00000000-0000-0000-0000-000000000000"
```
Comment (arielmelendez) [inline dockerfile]: Would it be problematic to use 1 and 2 for these to disambiguate the three of them?

Comment (agsuy) [reply dockerfile]: Nope, done

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**